include/rocksdb/listener.h

```diff
@@ -115,6 +115,7 @@ enum class FlushReason : int {
   kAutoCompaction = 0x09,
   kManualFlush = 0x0a,
   kErrorRecovery = 0xb,
+  kFlushNoSwitchMemtable = 0xc,
```

Comment:
The name might be a bit confusing as this is also during error recovery. Maybe we can call it something like ```kErrorRecoveryRetryFlush```?

Reply:
Sure, but we may be better to indicate that SwitchMemtable will not be called in this case at some place. Where do you think is appropriate to write the comment?

Reply:
The comment can be here.